README.md

```diff
@@ -22,21 +20,85 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+Running `rubycritic_extension -b base_branch, feature_branch` will analyse all the Ruby files in the base_branch and the feature_branch and compare the score between the two branches, before running this commit all your changes in both the branches:
+
+```bash
+$ rubycritic_extension -b base_branch,feature_branch
+```
+
+Alternatively if you're using GitLab you can pass `pull_request_id` as the third argument which will post a comment to the respective pull_request:
+
+```bash
+$ rubycritic_extension -b base_branch,feature_branch,pull_request_id
```

Comment:
On second thought `-b` and `-p` can be optional, since we can assume the extension does only two things -
1. runs rubycritic on two branches - hence first 2 args are branch names
2. optionally posts to PR - hence third arg optional pull_request_id